c2corg_ui/static/js/filters.js

```diff
@@ -16,3 +16,17 @@ app.trustAsHtmlFilter = function($sce) {
 };
 
 app.module.filter('appTrustAsHtml', app.trustAsHtmlFilter);
+
+
+/**
+ * @export
+ * @ngInject
```

Comment:
I don't think you need ``@ngInject`` here since there's nothing to inject in your filter.